### PR TITLE
remove redundant ToArray

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Comment by joelverhagen** [comment]
These tests are failing:
```
NuGetGallery.PackageOwnershipManagementServiceFacts+TheRemovePackageOwnerAsyncMethod.RemovingNamespaceOwnerRemovesPackageVerified [FAIL] NuGetGallery.PackageOwnershipManagementServiceFacts+TheRemovePackageOwnerAsyncMethod.MultipleNamespaceOwnersRemovalWorksCorrectly [FAIL]
NuGetGallery.PackageOwnershipManagementServiceFacts+TheRemovePackageOwnerAsyncMethod.RemovingNamespaceOwnerAsOrganizationAdminRemovesPackageVerified [FAIL] NuGetGallery.PackageOwnershipManagementServiceFacts+TheRemovePackageOwnerAsyncMethod.AdminCanRemoveAnyOwner [FAIL]
NuGetGallery.PackageOwnershipManagementServiceFacts+TheRemovePackageOwnerWithMessagesAsyncMethod.RemovingNamespaceOwnerRemovesPackageVerified [FAIL]
NuGetGallery.PackageOwnershipManagementServiceFacts+TheRemovePackageOwnerWithMessagesAsyncMethod.MultipleNamespaceOwnersRemovalWorksCorrectly [FAIL]
NuGetGallery.PackageOwnershipManagementServiceFacts+TheRemovePackageOwnerWithMessagesAsyncMethod.RemovingNamespaceOwnerAsOrganizationAdminRemovesPackageVerified [FAIL] NuGetGallery.PackageOwnershipManagementServiceFacts+TheRemovePackageOwnerWithMessagesAsyncMethod.AdminCanRemoveAnyOwner [FAIL]
NuGetGallery.PackageOwnershipManagementServiceFacts+TheRemovePackageOwnerWithMessagesAsyncMethod.AllowsNamespaceOwnershipToBeSkipped [FAIL]
```

Sample error:
```
    System.InvalidOperationException : Collection was modified; enumeration operation may not execute.
      Stack Trace:
           at System.Collections.Generic.HashSet`1.Enumerator.MoveNext()
           at NuGetGallery.PackageOwnershipManagementService.<RemovePackageOwnerImplAsync>d__19.MoveNext()
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
           at NuGetGallery.PackageOwnershipManagementService.<RemovePackageOwnerAsync>d__18.MoveNext()
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
           at NuGetGallery.PackageOwnershipManagementService.<RemovePackageOwnerAsync>d__17.MoveNext()
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
        tests\NuGetGallery.Facts\Services\PackageOwnershipManagementServiceFacts.cs(564,0): at NuGetGallery.PackageOwnershipManagementServiceFacts.TheRemovePackageOwnerAsyncMethod.<RemovePackageOwnerAsync>d__1.MoveNext()
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
        tests\NuGetGallery.Facts\Services\PackageOwnershipManagementServiceFacts.cs(669,0): at NuGetGallery.PackageOwnershipManagementServiceFacts.TheRemovePackageOwnerAsyncMethodFacts.<RemovingNamespaceOwnerRemovesPackageVerified>d__8.MoveNext()
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
        --- End of stack trace from previous location where exception was thrown ---
           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```

**Comment by SimonCropp** [comment]
test should be fixed

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "6.0.300",
```
Comment (erdembayar) [inline global.json:3]: Could we remove this change unless it's necessary for other changes? There're already exists another PR for this.

Comment (SimonCropp) [reply global.json]: done

**Review by erdembayar — APPROVED** [review]

**Pull request merged**